packages/unimodules-test-core/android/src/main/java/org/unimodules/test/core/BundleExtensions.kt

```diff
@@ -0,0 +1,3 @@
+package org.unimodules.test.core
+
+import android.os.Bundle
```

Comment:
What's the purpose of this file?

Reply:
Ping here? 🤔

Reply:
Sorry, it is a leftover from previous approach.